tests/testthat/test-derive_params_growth_height.R

```diff
@@ -39,3 +39,144 @@ test_that("derive_params_growth_heightlength Test 1: derive_params_growth_height
     filter(actual, PARAMCD == "WT2HTZ") %>% pull(AVAL)
   )
 })
+
+## Test 2: derive_params_growth_heightlength works for subjects that are under or over 2 years old
+## between visits
+test_that("derive_params_growth_heightlength Test 2: derive_params_growth_height works for subjects that are under or over 2 years old between visits", { # nolint
+  vs_data <- tibble::tribble(
+    ~USUBJID, ~SEX, ~HEIGHT, ~HEIGHTU, ~VSTESTCD, ~VSSTRESN, ~AAGECUR,
```

Comment:
Don't know if you meant to separate the over/under by gender as well, but it would be nice to maybe do 1 each?